### PR TITLE
Fix netperl compilation failure

### PR DESCRIPTION
**Pull request by Yingshun**
To compile netperl, the automake package needs to be installed
on source and target host.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
**Test result:**
_Before fix:_
```
# avocado run --vt-type libvirt --vt-machine-type q35 virsh.migrate_vm.positive_testing.live_migration.stress.host.netperf_network.with_postcopy --vt-connect-uri qemu:///system
JOB ID     : 8178943082c3a3e44f6464d556d4d9554a429324
JOB LOG    : /root/avocado/job-results/job-2020-07-13T21.03-8178943/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_vm.positive_testing.live_migration.stress.host.netperf_network.with_postcopy: ERROR: Packeage Error: Compile failed: Shell command failed: 'cd /var/tmp/  && tar jxf netperf-2.6.0.tar.bz2 && cd /var/tmp/netperf-2.6.0  && ./autogen.sh > /dev/null 2>&1 && ./configure --build=x86_64  > /dev/null 2>&1 && make > /dev/null 2>&1 '    (status: 127... (96.75 s)

```

_After fix:_
```
# avocado run --vt-type libvirt --vt-machine-type q35 virsh.migrate_vm.positive_testing.live_migration.stress.host.netperf_network.with_postcopy --vt-connect-uri qemu:///system

JOB ID     : eabaf56bc5c307f01bac44c64f1624eeb7f1a833
JOB LOG    : /root/avocado/job-results/job-2020-07-13T21.34-eabaf56/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_vm.positive_testing.live_migration.stress.host.netperf_network.with_postcopy: PASS (173.76 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 175.18 s

```

**Comment by Yingshun** [comment]
@chloerh  @dzhengfy

File: libvirt/tests/src/migration/migrate_vm.py [hunk 1]
```diff
@@ -797,7 +801,15 @@ def setup_netsever_and_launch_netperf(params):
...
     test_option = params.get("test_option", "")
     direction = params.get("direction", "remote")
-
+    remote_session = remote.remote_login("ssh", server_ip, "22", server_user,
+                                         server_pwd, r'[$#%]')
+    for loc in ['source', 'target']:
+        session = None
+        if loc == 'target':
+            session = remote_session
+        if not utils_package.package_install(["automake"], session):
```
Comment (chloerh) [inline libvirt/tests/src/migration/migrate_vm.py:810]: We are not forced to use [] when there's only 1 pkg to install.

Comment (Yingshun) [reply libvirt/tests/src/migration/migrate_vm.py]: Updated.

**Pull request merged**